src/plugins/Plugin.js

```diff
@@ -19,7 +19,7 @@ module.exports = class Plugin {
     this.opts = opts || {}
 
     // clear everything inside the target selector
-    this.opts.replaceTargetContent === this.opts.replaceTargetContent || true
+    this.opts.replaceTargetContent = this.opts.replaceTargetContent !== undefined ? this.opts.replaceTargetContent : true
```

Comment:
good catch! strange that the linter didn't catch this!

Reply:
I've had a quick search but couldn't find an eslint rule for this.